project/src/main/java/com/carlease/project/application/ApplicationServiceImpl.java

```diff
@@ -1,67 +1,111 @@
+    public List<ApplicationFormDto> findAll() {
+        List<Application> applications = applicationRepository.findAll();
+        return applications.stream()
+                .map(applicationMapper::toDto)
+                .toList();
     }
 
     @Override
-    public Application findById(long id) {
-        return applicationRepository.findById(id).orElseThrow();
+    public ApplicationFormDto findById(long id) throws ApplicationNotFoundException {
+        Application application = applicationRepository.findById(id).orElseThrow(() -> new ApplicationNotFoundException("id"));
+        return applicationMapper.toDto(application);
     }
 
     @Override
-    public Application create(ApplicationFormDto applicationFormDto) {
-
-        Application application = applicationMapper.applicationFormDtoToApplication(applicationFormDto);
+    public ApplicationFormDto create(ApplicationFormDto applicationFormDto) {
+        Application application = applicationMapper.toEntity(applicationFormDto);
 
-        String carMake = applicationFormDto.getCarMake();
-        String carModel = applicationFormDto.getCarModel();
+        Car car = carRepository.findByMakeAndModel(applicationFormDto.getCarMake(), applicationFormDto.getCarModel());
 
-        Car car = carRepository.findByMakeAndModel(carMake, carModel);
-        User user = userRepository.findById(applicationFormDto.getUserId()).orElseThrow(
-                () -> new RuntimeException("could not create user"));
+        User user = userRepository.findById(applicationFormDto.getUserId())
+                .orElseThrow(() -> new RuntimeException("Could not find user"));
 
         application.setCar(car);
         application.setUser(user);
 
-        application.setStatus(ApplicationStatus.PENDING);
+        application.setStatus(applicationFormDto.getStatus());
 
-        return applicationRepository.save(application);
+        Application savedApplication = applicationRepository.save(application);
+        return applicationMapper.toDto(savedApplication);
     }
 
     @Override
-    public List<Application> findAllByUserId(long id) {
-        return applicationRepository.findApplicationsByUserUserId(id);
+    public List<ApplicationFormDto> findAllByUserId(long id) {
+        List<Application> applications = applicationRepository.findApplicationsByUserUserId(id);
+        return applications.stream()
+                .map(applicationMapper::toDto)
+                .toList();
     }
 
+    public void evaluation(ApplicationFormDto applicationDto) {
+
+        InterestRateDTO interestRateDTO = interestRateService.findAll().getFirst();
+        InterestRate interestRate = interestRateMapper.toEntity(interestRateDTO);
+
+        CarPrice price = autosuggestorServiceImpl.carPrice(autosuggestorServiceImpl.calculateAverageCarPriceDependingOnYear(applicationDto, applicationDto.getManufactureDate()));
+        if (applicationDto.getStatus() == ApplicationStatus.PENDING) {
```

Comment:
```suggestion
        if (ApplicationStatus.PENDING.equals(applicationDto.getStatus())) {
```
nullsafe equality check unless you want to throw nullpointer when status is null